{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/settings.py

```diff
@@ -1,124 +1,167 @@
-from os import environ, getenv
+from os import environ
 from os.path import abspath, dirname, join
 from sys import argv
+
 import sentry_sdk
-from sentry_sdk.integrations.django import DjangoIntegration
 from configurations import Configuration
+from sentry_sdk.integrations.django import DjangoIntegration
 
 
-BASE_DIR = dirname(dirname(abspath(__file__)))
-PROJECT_NAME = '{{cookiecutter.project_name}}'
-PROJECT_ENVIRONMENT_SLUG = '{}_{}'.format(PROJECT_NAME, environ.get('DJANGO_CONFIGURATION').lower())
+PROJECT_NAME = "{{cookiecutter.project_name}}"
+CONFIGURATION = environ["DJANGO_CONFIGURATION"]
 
 # Detect if we are running tests.  Is this really the best way?
-IN_TESTS = 'test' in argv
+IN_TESTS = "test" in argv
+
+
+def get_env(name, default=None, required=False, cast=str):
+    """
+    Get an environment variable
+    Arguments:
+        name (str): Name of environment variable
+        default (Any): default value
+        required (bool): If True, raises an ImproperlyConfigured error if not defined
+        cast (Callable): function to call with extracted string value.
+            Not applied to defaults.
+    """
+
+    def _lookup(self):
+        value = environ.get(name)
+
+        if value is None and default is not None:
+            return default
+
+        if value is None and required:
+            raise ValueError(f"{name} not found in env")
+
+        return cast(value)
+
+    return property(_lookup)
+
+
+def csv_to_list(value):
+    """
+    Convert a comma separated list of values into a list.
+    Convenience function for use with get_env() and get_secret() ``cast`` argument.
+    """
+    if value is None:
+        return []
+    return value.split(",")
 
 
 class RedisCache(object):
```

Comment:
Lets trash this class and combine with the Dev settings.

Reply:
Can come after merge.

Reply:
Yeah, planning on going over the whole settings file in a separate PR/issue. Only did what was needed this one.

Reply:
Merge with Deployed class